apps/session-manager/src/utils/express.ts

```diff
@@ -8,6 +8,13 @@ import * as T from "fp-ts/Task";
 import { pipe } from "fp-ts/lib/function";
 import * as RTE from "fp-ts/ReaderTaskEither";
 
+export type ResLocals = Record<string, unknown> & {
+  // eslint-disable-next-line functional/prefer-readonly-type
+  detail?: string;
+  // eslint-disable-next-line functional/prefer-readonly-type
+  body?: Buffer;
```

Comment:
remove the linter exclusion if possible

Reply:
done in https://github.com/pagopa/io-auth-n-identity-domain/pull/22/commits/0aac1a7af5e32060657c60e53ef458e41e4e445c